frontend/packages/topology/src/behavior/usePanZoom.tsx

```diff
@@ -31,6 +36,9 @@ export const usePanZoom = (zoomExtent: [number, number] = ZOOM_EXTENT): PanZoomR
         if (node) {
           // TODO fix any type
           const $svg = d3.select(node.ownerSVGElement) as any;
+          if (node && node.ownerSVGElement) {
+            node.ownerSVGElement.addEventListener('mousedown', propagatePanZoomMouseEvent);
```

Comment:
Make sure to remove this listener in the dispose function.

Reply:
done